File: helixtrajectorycpp/src/cpp/drivetrain/HolonomicDrivetrain.cpp
// Copyright (c) TrajoptLib contributors

#include "drivetrain/HolonomicDrivetrain.h"

#include "drivetrain/Drivetrain.h"
#include "obstacle/Obstacle.h"

namespace helixtrajectory {

HolonomicDrivetrain::HolonomicDrivetrain(double mass, double momentOfInertia)
    : Drivetrain(mass, momentOfInertia) {}
}  // namespace helixtrajectory

File: java/src/main/java/org/sleipnirgroup/trajoptlib/HolonomicDrivetrain.java
// Copyright (c) TrajoptLib contributors

package org.sleipnirgroup.trajoptlib;

public abstract class HolonomicDrivetrain extends Drivetrain {
    protected HolonomicDrivetrain(double mass, double momentOfInertia, Obstacle bumpers) throws NullPointerException {
        super(mass, momentOfInertia, bumpers);
    }
}
